website/docs/r/vdc_group.html.markdown

```diff
@@ -51,6 +51,9 @@ The following arguments are supported:
   This is mainly useful when using
   [`vcd_nsxt_distributed_firewall_rule`](/providers/vmware/vcd/latest/docs/resources/nsxt_distributed_firewall_rule)
   resource as it cannot remove the default rule.
+* `force_delete` - (Optional, *3.11+*) When `true`, will force VDC Group deletion. It should clean
+  up child components. Default `false`. **Note:** when setting it to `true` for existing resource,
```

Comment:
I'm reading this and it's not very clear what happens with the default `false`. Is the VDC Group not deleted at all? Is it deleted only if there are no children and breaks otherwise? I think we should explain that.

Reply:
I have attempted to rephrase it. I wonder if it helps.

The `force_delete` is an API call flag. We did not support that before and VDC Group deletion could fail if there are remaining child components. Sending this flag cleans up child components automatically (the quality of this depends on VCD). The new test `TestAccVcdVdcGroupForceDeletion` actually validates that at least child IP Sets are in fact removed (https://github.com/vmware/terraform-provider-vcd/pull/1097/files#diff-a429766b6deabd8523c75d1a3e9a58033415f3ead79324fced4e2ab1203683e5R115-R120)